modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro

```diff
@@ -94,6 +107,23 @@ definition {
 		AppBuilderAdmin.gotoBack();
 	}
 
+	macro addFormViewSameFields {
+		while ( IsElementNotPresent(locator1 = "AppBuilder#NAVBAR") || IsElementNotPresent(locator1 = "FormViewBuilder#SIDEBAR_FIELDS_TAB") ) {
+			Pause(locator1 = "1000");
+		}
```

Comment:
Please, make sure if this part of code is really necessary. If not, you can remove it.